app/models/concerns/cfe_submission_state_machine.rb

```diff
@@ -62,8 +63,12 @@ module CFESubmissionStateMachine
         transitions from: :other_income_created, to: :irregular_income_created, guard: :non_passported?
```

Comment:
Should this transition to `employments_created`?

Reply:
No, I don't think so.  It's the `irregular_income_created` event, so it transitions to `irregular_income_created`.  The `employments_created` event transitions to `employments_created`.

Reply:
Yes, I see it now! :+1: